build.zig

```diff
@@ -17,10 +17,54 @@ pub fn build(b: *std.Build) !void {
         },
     };
 
+    const c_sdl_preferred_linkage = b.option(
+        std.builtin.LinkMode,
+        "c_sdl_preferred_linkage",
+        "Prefer building statically or dynamically linked libraries (default: static)",
+    ) orelse .static;
+    const c_sdl_strip = b.option(
+        bool,
+        "c_sdl_strip",
+        "Strip debug symbols (default: varies)",
+    ) orelse (optimize != .Debug);
+    const c_sdl_sanitize_c = b.option(
+        enum { off, trap, full }, // TODO: Change to std.zig.SanitizeC after 0.15
+        "c_sdl_sanitize_c",
+        "Detect C undefined behavior (default: varies)",
+    ) orelse .off;
```

Comment:
```suggestion
    ) orelse switch (optimize) {
        .Debug => .full,
        .ReleaseSafe => .trap,
        .ReleaseFast, .ReleaseSmall => .off,
    };
```

https://github.com/ziglang/zig/blob/e96d86064eb81977f254fa8f36481b7d150cb3b6/src/Package/Module.zig#L238-L250